Description: Use the correct BSD return types for uum ttyfdslot.
 Source commits: f8e06ee3e8565c6d5508fd2bed614441fa39b7e8
Origin: upstream
Forwarded: yes
Last-Update: 2026-07-13

diff --git a/Wnn/uum/ttyfdslot.c b/Wnn/uum/ttyfdslot.c
index e5f0b4f..344860d 100644
--- a/Wnn/uum/ttyfdslot.c
+++ b/Wnn/uum/ttyfdslot.c
@@ -112,7 +112,7 @@ ttyfdslot (int fd)
   register int slotnum;
   register struct ttyent *te;
   if ((fullnamp = ttyname (fd)) == NULL)
-    return NULL;
+    return 0;
   if ((ttynamp = rindex (fullnamp, '/')) == NULL)
     {
       ttynamp = fullnamp;
@@ -130,7 +130,7 @@ ttyfdslot (int fd)
         }
     }
   endttyent ();
-  return NULL;
+  return 0;
 }
 
 #endif /* defined(BSD43) && !defined(linux) */
